src/server.ts

```diff
@@ -2,12 +2,13 @@ import * as assert from 'assert';
 import safeNavigation from '@smartlyio/safe-navigation';
 import { Make, Maker, ValidationError, validationErrorPrinter } from './make';
 
-export interface Response<Status extends number, ContentType, Value> {
+export interface Response<Status extends number, ContentType, Value, Headers = unknown> {
   status: Status;
   value: {
     contentType: ContentType;
     value: Value;
   };
+  headers: Headers;
```

Comment:
hm, making `headers` not-optional here means we'll need to generate something for it in `oats`...